Configure

```diff
@@ -2714,6 +2719,19 @@ exit(0);
 # Helpers and utility functions
 #
 
+# Death handler, to print a helpful message in case of failure #######
+#
+sub death_handler {
+    my $build_file = $target{build_file} // "build file";
+    print STDERR <<"_____";
+
+Failure!  $build_file wasn't produced
+Please read INSTALL and associated NOTES files, look over your available
+compiler tool chain or change your configuration.
```

Comment:
On some level it's a bit misleading, because it kind of suggests that there are alternatives to reading INSTALL and NOTES :-)